dtfs-central-api/src/v1/controllers/utilisation-report-service/post-add-payment.controller/index.test.ts

```diff
@@ -0,0 +1,158 @@
+import httpMocks from 'node-mocks-http';
+import { ObjectId } from 'mongodb';
+import { ApiError, Currency } from '@ukef/dtfs2-common';
+import { HttpStatusCode } from 'axios';
+import { PostAddPaymentRequest, postAddPayment } from '.';
+import { TfmSessionUser } from '../../../../types/tfm/tfm-session-user';
+import { aTfmSessionUser } from '../../../../../test-helpers/test-data/tfm-session-user';
+import { addPaymentToUtilisationReport } from './helpers';
+import { PostAddPaymentPayload } from '../../../routes/middleware/payload-validation/validate-post-add-payment-payload';
+import { NewPaymentDetails } from '../../../../types/utilisation-reports';
+
+jest.mock('./helpers');
+
+class TestApiError extends ApiError {
```

Comment:
Not sure if this is the best approach to do this, but this allows us to test that errors which extend `ApiError` are handled correctly rather than needing the rely on a specific class which could be removed down the line